iot_api/user_api/models/NotificationAssetTag.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, BigInteger
+from sqlalchemy.sql.schema import ForeignKey
+from iot_api.user_api import db
+
+class NotificationAssetTag(db.Model):
+    __tablename__ = 'notification_asset_tag'
+    user_id = Column(BigInteger, ForeignKey("iot_user.id"), nullable=False, primary_key=True)
+    tag_id = Column(BigInteger, ForeignKey("tag.id"), nullable=False, primary_key=True)
+    
+    @classmethod
+    def find_all_by_user_id(cls, user_id):
```

Comment:
The "by_user_id" part in the name of the function is quite repetitive when the function is called:

`find_all_by_user_id(user_id = user_id)`

I think it is better to indicate the filtering parameters using the parameters name, i.e:

`find_all_with(user_id = user_id)`

This also allows shorter function names when they have many filtering parameters.

Reply:
Done